cli/dcoscli/log.py

```diff
@@ -351,6 +352,8 @@ def print_logs_range(url):
     with contextlib.closing(http.get(url, is_success=is_success,
                                      headers={'Accept': 'text/plain'})) as r:
 
+        if r.status_code == 500:
+            raise DCOSUnprocessableException(r)
```

Comment:
The class pydoc states `A wrapper around Response objects for HTTP 422 error codes`, so this seems not suited for a 500 response.

What about adding a `text()` method to `DCOSHTTPException`, and if it's a 500 raise a `DCOSException` with the custom message you want (which would include the response content)?

Reply:
what if i declare a new exception inherited from `DCOSHTTPException` and override `__str__` method to include text? would that work?

Reply:
Creating an exception class for 500 error seems fine, though I don't see why the string representation should vary depending on the class. Eg. why would a 500 error print the response content by default while a 502 wouldn't? That seems arbitrary.

I think we should expose the response content from DCOSHTTPException in all cases, and leave it to the caller if they want to print some customized/augmented error messages.